tensorflow/java/src/gen/resources/tftypes.csv

```diff
@@ -0,0 +1,21 @@
+TF type,datatype code,Java type,Java boxed type,Creator?,default,Description
+Float,1,float,Float,y,0f,32-bit single precision floating point number
+Double,2,double,Double,y,0,64-bit double precision floating point number
+Int32,3,int,Integer,y,0,32-bit signed integer
+UInt8,4,byte,Byte,n,,8-bit unsigned integer
+Int16,5,,Short,n,(short)0,16-bit signed integer

```

Comment:
I'm pretty sure that the 'datatype code' is no longer needed, so shall we remove that.
I'm also unclear about whether the boxed type is needed. Is it? 
And perhaps use 'zero value' instead of 'default' for the header line?